src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -373,7 +373,7 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
         scrolled={true}
       >
         <Grid container item xs={12} spacing={2} textAlign='left'>
-          <Grid xs={12} padding={theme.spacing(1, 3, 1, 5)}>
+          <Grid item xs={12} padding={theme.spacing(1, 3, 1, 5)}>
```

Comment:
Adding this "item" is making all rows to be at the same level of the grey box of observations. We should add more padding to this rows

Reply:
Yes good point, I will revisit this.

Reply:
cc @joshbuchea